src/main/java/net/datafaker/providers/base/Number.java

```diff
@@ -74,6 +78,7 @@ public long numberBetween(long min, long max) {
         return decimalBetween(realMin, realMax).longValue();
     }
 
+
```

Comment:
why do we need this line?

Reply:
> why do we need this line?

Should I just call, isValidRange() and also make a a function isValidRange(long, long)? I tried doing it but the build failed.

Reply:
>I tried doing it but the build failed.

in fact you didn't . If you look at your old changes there was a method for `float` but not for `long` that's the problem
https://github.com/datafaker-net/datafaker/pull/739/files#diff-94330e19cbdfbd84d55b982b84194e54d5a83486d1c78ee333fcaa76b5238ac3R84

Reply:
> > I tried doing it but the build failed.
> 
> in fact you didn't . If you look at your old changes there was a method for `float` but not for `long` that's the problem https://github.com/datafaker-net/datafaker/pull/739/files#diff-94330e19cbdfbd84d55b982b84194e54d5a83486d1c78ee333fcaa76b5238ac3R84
oh yes, i completely missed it! thank you for looking into it. fixed.